app/components/explore/side-bar.js

```diff
@@ -44,6 +44,10 @@ export default Component.extend({
       this.send('selectDateFilter', 'custom_dates');
     },
 
+    selectEventCfs(cfs) {
```

Comment:
Just out of doubt, What is this property being used for ?

Reply:
to set `cfs` variable to either `open` or `closed`. Via this, events are filtered,